ledis/Controllers/SignUpViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  SignUpViewController.swift
+//  ledis
+//
+//  Created by Maya Vu on 13.05.18.
+//  Copyright © 2018 Maya Vu. All rights reserved.
+//
+
+import UIKit
+import Firebase
+
+class SignUpViewController: UIViewController {
+    
+    @IBOutlet weak var ucheckedBox: UIButton!
+    @IBOutlet weak var usernameTextField: UITextField!
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var signUpButton: UIButton!
+    
+    var uncheckedBox = UIImage(named: "elipse")
+    var checkedBox = UIImage(named: "CheckBox1")
+    var isBoxClicked = Bool()
```

Comment:
it would be good to encapsulate all logic of this uncheckedBox into separate custom class of UIButton